spk/python310/Makefile

```diff
@@ -39,18 +39,18 @@ WHEELS = src/requirements-crossenv.txt src/requirements-pure.txt src/requirement
 ##
 
 # [gevent]
-# DEPENDS += cross/libev cross/c-ares
-# ENV += GEVENTSETUP_EMBED_CARES=FALSE
-# ENV += GEVENTSETUP_EMBED_LIBEV=FALSE
+DEPENDS += cross/libev cross/c-ares
```

Comment:
Same question, why are these sometimes on or off?

Reply:
Same question but different answer...  By default python310 comes with no wheels (besides certificates).  Including wheels in python package ends-up breaking individual packages dependent on python310 as they cannot superseed the version provided by default with python310.  As such it was decided to not be including any and leave all of them in the sub-application packages.

But, there are many caveats at cross-compiling wheels.  As such I've included all the most common ones in python310 to test and demonstrate that cross-compiling do work at every updates.  This allows me to switch then ON then OFF simply to confirm builds are succesfull.  Application packages can then refer to python310 as it hosts the most common combinations and can be used as an example.

Hope this helps.